views/subscription-deferred-install-approval-form.hbs

```diff
@@ -44,7 +44,7 @@
             <section>
               <div>
                 <h2>The request has been approved.</h2>
-                <p><a href="{{{orgUrl}}}" target="_blank">{{orgName}}</a> was successfully connected to "<a href="{{{jiraHost}}}" target="_blank">{{jiraHost}}</a>.</p>
+                <p><a href="{{{orgUrl}}}" target="_blank">{{orgName}}</a> was successfully connected to <a href="{{{jiraHost}}}" target="_blank">{{jiraHost}}</a>.</p>
                 <p>You can close the window now.</p>
```

Comment:
Nit: should this be window or tab?

Reply:
Whatever you prefer, @rachellerathbone :) You saw my preferences in my previous PR